doc/getting-started/getting-started.rst

```diff
@@ -154,7 +154,7 @@ To set up the ACRN build environment on the development computer:
 
    .. code-block:: bash
 
-      sudo pip3 install lxml xmlschema defusedxml
+      sudo pip3 install lxml xmlschema defusedxml tqbm
```

Comment:
@NanlinXie @junjiemao1 Can we used one of the requirements.txt files instead of listing python packages here in the documentation?  We've got these two in the tree now, but neither has this same list of packages shown in the GSG.
```
~/projectacrn/acrn-hypervisor$ more ./misc/config_tools/requirements.txt
defusedxml
lxml
elementpath
xmlschema
~/projectacrn/acrn-hypervisor$ more ./misc/config_tools/configurator/requirements.txt
build
tqdm
requests
lxml
xmltodict
sphinx
bs4
```

Reply:
That means  we should add a new command to clean configurator other than 'make clean'.

Reply:
I'm not sure the connection to "make clean" here?

Reply:
@junjiemao1 I thought the plan was to merge these two requirements.txt into one (config_tools/requirements.txt) and then we can reference that combined requirements.txt in the GSG  @amyreye

Reply:
> I'm not sure the connection to "make clean" here?

Because 'make clean' also clear configurator. If  'tqdm' not installed, 'make clean' may emit warning.
So 'tqdm' only installed when make configurator,  we need add a command like 'make clean-conifgurator'.

Reply:
When we do a "make clean", we don't expect to clean any apt or python packages that were installed on the system.  We're only trying to clean the  directories in the repo to remove any build artifacts that could impact another build.

I'd be OK with merging this PR and dealing with the two requirements.txt files later...

Reply:
"make clean"  use python to clear file list and this file import tqdm.  It will not clean apt or python packages.

Reply:
> 
> I'd be OK with merging this PR and dealing with the two requirements.txt files later...
As misc/config_tools/configurator/README:
> cd acrn-hypervisor/misc/config_tools
> python3 -m pip install -r requirements.txt
> cd configurator
> python3 -m pip install -r requirements.txt

These two files is not only doc files. They also used in configuration build.
 User can build configurator by misc/config_tools/configurator/README.md on windows without read GSG.
If remove this package in requirements, it's inconvenient for windows user.

Reply:
> "make clean" use python to clear file list and this file import tqdm. It will not clean apt or python packages.

Am I understanding this correctly that running these scripts will install a bunch of Python tools and dependencies in the main source tree?

This may be part of a discussion that I missed earlier but why do we have a "third-party library manager", couldn't we handle installing/checking/cleaning these dependencies relying on the user (i.e. using `pip3 install...`)?

Reply:
I think it's a shortcut for user to save time. For emaple a Jenckins server maintainer can setup a server quickly.

Reply:
GSG is a guide document, and priovide a shortcut for user to set build enviroment.
'requirements.txt' is a script for both linux and windows.
They are independent.

'make clean' clean files by  'python ./configurator/packages/configurator/thirdLib/manager.py clean'. 
So 'tqdn' need to add to the GSG.

For 'requirements.txt' script is used by both linux and windows, I think it shouldn't be modified.
How about your opinion？ @dbkinder @gvancuts

Reply:
My request is to merge the two requirements.txt files, and then use that combined requirements.txt file in the GSG instead of having a separately maintained list in the GSG.

Reply:
> GSG is a guide document, and priovide a shortcut for user to set build enviroment. 'requirements.txt' is a script for both linux and windows. They are independent.

The `requirements.txt` file is there to help make the GSG easier to go through, the user can run `pip3 install -r requirements.txt` in one command instead of manually listing all the Python packages that need to be installed. It also allows us to indicate if there are preferred or minimum versions for certain packages.

> 
> 'make clean' clean files by 'python3 ./configurator/packages/configurator/thirdLib/manager.py clean'. So 'tqdn' need to add to the GSG.

That's understood, what I do not understand is why we have this thirdLib manager?

> 
> For 'requirements.txt' script is used by both linux and windows, I think it shouldn't be modified. How about your opinion？ @dbkinder @gvancuts

I'm not sure I understand why you say that this file shouldn't be modified? If the Python dependencies or requirements change, we should absolutely update this file to reflect that. What @dbkinder is to have a single `requirements.txt` to make it easier to maintain it and easier for the user (running `pip3 install -r ...` only once instead of twice, and we only need to care about a single file to have all dependencies in it.

Reply:
The two requirements.txt files merged. @dbkinder

Reply:
> This may be part of a discussion that I missed earlier but why do we have a "third-party library manager", couldn't we handle installing/checking/cleaning these dependencies relying on the user (i.e. using `pip3 install...`)?

@NanlinXie @junjiemao1 , can you help sched some light on this?